profanity.rb

```diff
@@ -51,9 +51,10 @@ def self.log_file
     return File.open(LOG_FILE, 'a') { |file| yield file } if block_given?
   end
 
+  @opts = Opts.new(ARGV)
   @title  = nil
   @status = nil
-  @char   = Opts.char.capitalize
+  @char   = @opts.char.capitalize
```

Comment:
should only use instance variables in a class but it's fine for now until we circle back to refactor this file